src/tests/gov/nasa/jpf/StaticCallToNonStaticTest.java

```diff
@@ -0,0 +1,35 @@
+package gov.nasa.jpf;
+
+import org.junit.AfterClass;
+import gov.nasa.jpf.util.test.TestJPF;
+import org.junit.jupiter.api.Test;
+import java.io.File;
+
+
+
+
+public class StaticCallToNonStaticTest extends TestJPF {
+
+    @Test
+    public void testStaticCallToNonStatic() {
+  D.m();
+}
+
+    // Cleanup method to delete the temporary file after tests
+    @AfterClass
```

Comment:
Is this cleanup function needed? Your new test does not create a temp. file directly. If this happens due to some side effect somewhere, we can ignore it (and add that file to `.gitignore`).